### PR TITLE
Move ad-block-updater packaging into its own script

### PR DESCRIPTION
**Pull request by antonok-edm**
Many of the other packaging scripts have their own logic in separate files. There is planned work on the adblock component packaging; separating this out into its own step will make it much easier to upgrade in the future.

### CODE REVIEW

File: scripts/packageAdBlock.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```
[the remark this answers is not in the document]
Comment (antonok-edm) [reply scripts/packageAdBlock.js:1]: Yeah, there are only 4 files which actually _have_ it in this repo. I added it here at least but that might be worth revisiting later.

File: scripts/packageAdBlock.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Example usage:
+//  npm run package-ad-block -- --binary "/Applications/Google\\ Chrome\\ Canary.app/Contents/MacOS/Google\\ Chrome\\ Canary" --key-file path/to/ad-block-updater-regional-component-keys
```
[the remark this answers is not in the document]
Comment (antonok-edm) [reply scripts/packageAdBlock.js:6]: I _think_ Brave should work fine here too

File: scripts/packageComponent.js [hunk 3]
```diff
@@ -192,8 +138,6 @@ const getDATFileListByComponentType = (componentType) => {
     case 'speedreader-updater':
       return [path.join('node_modules', 'speedreader', 'data', 'speedreader-updater.dat'),
         path.join('node_modules', 'speedreader', 'data', 'content-stylesheet.css')]
-    default:
```
[the remark this answers is not in the document]
Comment (antonok-edm) [reply scripts/packageComponent.js:195]: updated

File: scripts/packageComponent.js [hunk 4]
```diff
@@ -103,54 +82,33 @@ const getDATFileVersionByComponentType = (componentType) => {
...
   switch (componentType) {
     case 'ethereum-remote-client':
       return path.join('node_modules', 'ethereum-remote-client')
     case 'wallet-data-files-updater':
       return path.join('node_modules', 'brave-wallet-lists')
-    case 'ad-block-updater':
-      return path.join('build', 'ad-block-updater')
     case 'https-everywhere-updater':
     case 'local-data-files-updater':
       // TODO(emerick): Make these work like ad-block
```
[the remark this answers is not in the document]
Comment (antonok-edm) [reply scripts/packageComponent.js]: yep, good idea - done

**Review by fmarier — APPROVED** [review]

**Pull request merged**